src/app/in-memory-data.service.ts

```diff
@@ -1,7 +1,7 @@
 import { InMemoryDbService } from 'angular2-in-memory-web-api';
 export class InMemoryDataService implements InMemoryDbService {
   createDb() {
-    let workItems = [
+    let workitems = [
```

Comment:
I also would, but the name gets reflected as the url component and thus must be compatible with the "real" api.